libkbfs/block_journal.go

```diff
@@ -895,6 +901,16 @@ func (j *blockJournal) removeFlushedEntry(ctx context.Context,
 		}
 	}
 
+	// Remove the corresponding block from disk.  TODO: eventually we
```

Comment:
I think it's cleaner to leave the initial part of this function as is, and have something like this here:

```
if entry.Op == putOp || entry.Op == addRefOp {
  id, _, err := ...
  if err != nil {
    ...
  }

  if !j.hasRef(id) {
    ...
  }
}
```